indexer/variables.tf

```diff
@@ -472,3 +472,15 @@ variable "image_count" {
   description = "Number of images to store for ECR, defaults to 100."
   default     = 100
 }
+
+variable "create_read_replica_2" {
+  description = "Create read replia 2 or not. Default: true"
+  type        = bool
+  default     = true
+}
+
+variable "enable_rds_main_multiaz" {
+  description = "Enable RDS main instance. Default: true"
+  type        = bool
+  default     = true
+}
```

Comment:
would there be a case where the values of these will be different? for example, 

`create_read_replica_2 = true, enable_rds_main_multiaz = false`?

if these two values will stay the same, then would it make sense to combine them into a single variable?

Reply:
Yes. It could be possible to be different. For example, if on dev, you want to test the load distribution between read replicas then you should let `create_read_replica_2` = true. `enable_rds_main_multiaz` should be `false` on dev because that's wasted